agent/xds/delta_test.go

```diff
@@ -421,6 +411,153 @@ func TestServer_DeltaAggregatedResources_v3_BasicProtocol_HTTP2(t *testing.T) {
 	}
 }
 
+func TestServer_DeltaAggregatedResources_v3_SlowEndpointPopulation(t *testing.T) {
```

Comment:
This new test artificially slows down endpoint replies to try and force the bug. If this test is executed directly on `main` it fails while trying to unsubscribe from `geo-cache`.